### PR TITLE
Refine README with cleanup and editing

### PR DESCRIPTION
**Pull request by prestonso**
Some cleanup and editing to make the README a bit clearer to new users. Also added the Reservoir logo directly to README for consistent branding.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,34 @@
...
-[Composer-based project template][template]. 
-
-It's easy!
+[Composer-based project template][template]. It's easy!
 
-1. [Install Composer][getcomposer]
-
-2. Create a Reservoir project with `composer`
 ```
-$ composer create-project acquia/reservoir-project MY_PROJECT --stability=alpha
```
Comment (tedbow) [inline README.md]: This should not be removed. "--stability=alpha" is needed to work

File: README.md [hunk 2]
```diff
@@ -1,39 +1,34 @@
...
-# How to install Reservoir
+## How to install Reservoir
 
 The preferred way to install Reservoir is using our
-[Composer-based project template][template]. 
-
-It's easy!
+[Composer-based project template][template]. It's easy!
 
-1. [Install Composer][getcomposer]
```
Comment (tedbow) [inline README.md]: Did you mean to remove these steps?

File: README.md [hunk 3]
```diff
@@ -1,39 +1,34 @@
...
-
-2. Create a Reservoir project with `composer`
 ```
-$ composer create-project acquia/reservoir-project MY_PROJECT --stability=alpha
+$ composer create-project acquia/reservoir-project MY_PROJECT
 ```
-3. In your webhost configuration point my-project.com to [MY_PROJECT]/docroot
```
Comment (tedbow) [inline README.md]: Deleted on purpose?

File: README.md [hunk 4]
```diff
@@ -75,15 +71,13 @@ You'll want to:
...
+- [Add authentication information to the OpenAPI docs](https://www.drupal.org/node/2886726)
 - Add maintainable mechanism to not expose certain content/config entity types in both API docs nor allow accessing them via JSON API.
-- Ensure only users with the 'content_administrator' or 'client_developer' roles can log in.
-- Allow users with the 'client_developer' to create and edit roles other than 'client_administrator' and 'client_developer', and allow them only to grant+revoke non-restricted permissions. This then allows them to define roles (scopes) for clients.
+- Ensure only users with the `content_administrator` or `client_developer` roles can log in.
+- Allow users with the `client_developer` to create and edit roles other than `client_administrator` and `client_developer`, and allow them only to grant+revoke non-restricted permissions. This then allows them to define roles (scopes) for clients.
 - Make it easy to delete all default content in one go: node 1, users 2, 3 and 4.
 
 [template]: https://github.com/acquia/reservoir-project "Composer-based project template"
-[getcomposer]: https://getcomposer.org/ "Get Composer website"
```
Comment (tedbow) [inline README.md]: This needed if we add install steps back

**Review by tedbow — CHANGES_REQUESTED** [review]

**Comment by prestonso** [comment]
Looks like I was accidentally relying on an older version of this file. Stand by!

**Review by tedbow — APPROVED** [review]
Looks good

**Pull request merged**